liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -959,4 +854,102 @@ public void info(String msg) {
 
     }
 
+    private Plugin getLibertyPluging() {
+        Plugin plugin = project.getPlugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID + ":" +LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID);
+        if (plugin == null) {
+            plugin = plugin(LIBERTY_MAVEN_PLUGIN_GROUP_ID, LIBERTY_MAVEN_PLUGIN_ARTIFACT_ID,  "LATEST");
+        }
+        return plugin;
+    }
+
+    private Xpp3Dom getLibertyPluginConfig() {
+        // get the Liberty plugin configuration from the pom and overrides looseApplication to true.
+        // ignores the plugin execution configuration just like CLI invocation (e.g. mvn liberty:create) ignoring execution configuration.
+        Plugin libertyPlugin = getLibertyPluging();
+        Xpp3Dom overrides = configuration(element(name("looseApplication"), "true"));
```

Comment:
looseApplication now defaults to true. This is not needed.

Reply:
In case user sets it to false in the pom and Dev mode has to run with looseApplication=true.

Reply:
Should we log a message here letting the user know we overrode the looseApplication setting (if it was false).